### DISCUSSION TITLE
Editing Grid Builder element

### DISCUSSION THREAD

**Discussion by upperhouse** [awcodes/filament-tiptap-editor#351] [Q&A]
Hello.
Wondering if there is anyway, that maybe I am missing, to edit an existing grid element created with the Guid Builder tool, without resorting to editing the html manually. For instance removing an element in the grid to switch it from 3 columns to 2, or alternately adding additional columns.
Thanks.

**Comment by awcodes** [comment]
Currently there is not.

Easiest thing to do is just create a new grid then copy / paste or drag the content into the new grid.

**Comment by upperhouse** [comment]
That is what I thought. Thanks for your reply.

**Comment by GeorgeNance** [comment]
Is there any plans to make the grid area editable? This seems like a pretty important function for a WYSIWYG editor

**Reply by awcodes** [reply]
I don't disagree with you, but this would be a challenging thing to accomplish. Nor have I had the time to find a solid way to make it work.

It isn't out of the question though. So, maybe in the future at some point.
